src/reducers/posts.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+import {PostsTypes as types} from 'constants';
+
+const initState = {
+    status: 'not fetched',
+    error: null,
+    data: []
+};
+
+export default function reducePosts(state = initState, action) {
```

Comment:
Nothing to change here yet, but we're going to need to reorganize the store still to fit what we discussed last week. Ideally, the response to this call should add these posts to a map in `store.entities.posts.posts` and then their ids into `store.entities.posts.postIdsByChannelIds`. The status/error state will also need to be moved out into `store.requests`.